dotCMS/src/main/java/com/dotcms/contenttype/model/field/layout/FieldUtil.java

```diff
@@ -236,6 +239,15 @@ public static Field copyField(final Field field, final int newSortOrder) {
         return jsonFieldTransformer.from();
     }
 
+    /**
+     * Apply Internationalization to field property using a Language Variable, for each field property set into
+     * fieldMap a Language Variable with the name contentTypeVariable.fieldVariable.propertyName is searched, if it
+     * exists then the field's property value is replaced by the language variable value
+     *
+     * @param contentType
+     * @param contentTypeInternationalization set the mode, language and user to search the Language Variable
+     * @param fieldMap field properties
+     */
     public static void setFieldInternationalization(
```

Comment:
This method sounds like a good candidate to be moved to an API. Too much for a util method?

Reply:
done https://github.com/dotCMS/core/pull/18281/commits/c5d753c9f171f339c058cfc4df2a6d8a16a5725a